feat: support application level extensions (#9923)
argoproj/argo-cd: docs/developer-guide/ui-extensions.md

})(window)
```

## Application Tab Extensions

> The Argo CD Application is a Kubernetes resource so application tabs can be the same way as any other resource tab.

Reviewer: ```suggestion
Since the Argo CD Application is a Kubernetes resource, application tabs can be the same as any other resource tab.
```

small nitpicky grammar correction so it's easier to read

Author: thank you!
